### PR TITLE
Add db reset

### PR DESCRIPTION
**Pull request by LoriKarikari**
### Type: feature

Closes: [#241](https://github.com/blitz-js/blitz/issues/241)

### What are the changes and their implications? :gear:

 Added the `blitz db reset` command which deletes the database and runs a fresh migration afterwards. It works with PostgreSQL, MySQL and  SQLite.

### Checklist

- [ ] Tests added for changes
- [ ] Any added terminal logging uses `packages/server/src/log.ts`

### Other information
Since Prisma 2 doesn't have a `reset` command anymore I had to write some custom code to interact with the database. Currently this is done by getting the database url from the .env file and connecting with the database clients (except for SQLite). This currently works if the user sticks to the default options like using the public schema in Postgres and storing the SQLite database in the db folder. This can be optimized to use other schemas and database locations. Also maybe add a --force flag to skip the prompt.

### CODE REVIEW

**Comment by LoriKarikari** [comment]
> 😬 It seems like your prettier messed up reading configs, try rerunning it

Sorry, my bad. Opening VSCode from WSL messed with my plugins. Should be fixed now.

File: packages/cli/package.json [hunk 1]
```diff
@@ -1,80 +1,85 @@
...
+	},
+	"keywords": [
+		"blitz",
+		"cli"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/blitz-js/blitz"
+	},
+	"gitHead": "9c54cfaef19b63c30a7e41447541c7cc30b5c6d4"
```
Comment (flybayer) [inline packages/cli/package.json:84]: Sorry for the delay on review here!

This pkg.json is converted to tabs. Looks like this is still left over from the prettier mess up :)

**Comment by flybayer** [comment]
Overall this looks great!!

Did you know that prisma has `db.raw('select * ...')` for SQL access? 😬 I think you can remove the pg and mysql clients and use that.

**Comment by flybayer** [comment]
(converting to draft until ready for review again)

**Comment by LoriKarikari** [comment]
@flybayer Must have missed that, still learning a lot about Prisma. Will edit this and remove the clients.

**Comment by flybayer** [comment]
Sorry about that — I should have added this into the original issue description

**Comment by wKovacs64** [comment]
Closing in favor of #339

**Pull request closed without merging**